contrib/raftexample/README.md

```diff
@@ -0,0 +1,67 @@
+# raftexample
+
+raftexample is an example usage of etcd's raft library. It provides a simple REST API for a key-value store cluster backed by raft consensus.
+
+## Getting Starteed
+
+### Running raftexample
+
+First start a single-member cluster of raftexample:
+
+```sh
+raftexample --id 1 --cluster http://127.0.0.1:12379 --port 12380
+```
+
+Each raftexample process maintains a single raft instance and an optional key-value server.
+The process's list of comma separated peers (--cluster), its raft ID index into the peer list (--id), and http key-value server port (--port) are passed through the command line.
+
+Next, store a value ("hello") to a key ("my-key"):
+
+```
+curl -L http://127.0.0.1:12379/my-key -XPUT -d hello
+```
+
+Finally, retrieve the stored key:
+
+```
+curl -L http://127.0.0.1:12379/my-key
+```
+
+
+### Running a local cluster
+
+First install [goreman](https://github.com/mattn/goreman), which manages Procfile-based applications.
+
+The [Procfile script](./Procfile) will set up a local example cluster. You can start it with:
+
+```sh
+goreman start
+```
+
+This will bring up three raftexample instances.
+
+You can write a key-value pair to any member of the cluster and likewise retrieve it from any member.
+
```

Comment:
I tried it with my fixed goremon but kv recovery looks busted; will fiddle with it